relations/Cargo.toml

```diff
@@ -12,25 +12,28 @@ description = "An instantiation of the Poseidon SNARK-friendly hash function."
 
 [dependencies]
 ark-bls12-381 = { version = "^0.3.0", default-features = false }
-ark-crypto-primitives = { version = "^0.3.0", default-features = false, features = [ "r1cs" ] }
-ark-ec = { version = "^0.3.0", default-features = false }
-ark-ed-on-bls12-381 = { version = "^0.3.0", features = ["r1cs"] }
 ark-ff = { version = "^0.3.0", default-features = false }
-ark-gm17 = { version = "^0.3.0", default-features = false }
-ark-groth16 = { version = "^0.3.0", default-features = false }
-ark-marlin = { version = "^0.3.0", default-features = false }
-ark-poly = { version = "^0.3.0", default-features = false }
-ark-poly-commit = { version = "^0.3.0", default-features = false }
-ark-r1cs-std = { version = "^0.3.0", default-features = false }
 ark-relations = { version = "^0.3.0", default-features = false }
 ark-serialize = { version = "^0.3.0", default-features = false }
-ark-snark = { version = "^0.3.0", default-features = false }
 ark-std = { version = "^0.3.0", default-features = false }
-blake2 = { version = "0.9.2", default-features = false }
-liminal-ark-poseidon = { version = "0.1.0", default-features = false, features = ["circuit"] }
-liminal-ark-relation-macro = { version = "0.1.1" }
+liminal-ark-poseidon = { version = "0.1.0", default-features = false }
+# todo: remove path after publishing
+liminal-ark-relation-macro = { version = "0.1.2", path = "src/proc_macro" }
 paste = { version = "1.0.11" }
 
+# Required for computation in a circuit.
+ark-crypto-primitives = { version = "^0.3.0", default-features = false, features = [ "r1cs" ], optional = true }
```

Comment:
Nit: perhaps in the future we could have a uniform convention by either using `features = ["dependency_feature"]` or `"dependency/feature"` in the `features` table? There might be some intricacies I am not seeing here, but it would be nice to have dependency features defined in one way.

Reply:
@pmikolajczyk41 Feel free to resolve this at any time.

Reply:
fair point, but I would set it up on some retro/general forum and then adjust all crates to the decision